include/pini/pini.hpp

```diff
@@ -16,6 +17,13 @@ class pini {
 	bool load_file(std::filesystem::path const& filename);
 	bool load_text(std::string_view text);
 
+	// iteration
+
+	using const_iterator = std::unordered_map<std::string, std::string>::const_iterator;
```

Comment:
Just curious: any reason you don't want to alias the map type itself, and instead write down that long type name every time?

Reply:
I see that you do have the alias as `map_type`, so why not use that here?!